build.gradle

```diff
@@ -17,7 +17,7 @@ plugins {
 	id 'org.sonarqube' version '3.3'
 	id 'org.springframework.boot' version '2.4.12'
 	id "org.flywaydb.flyway" version '8.5.2'
-	id 'au.com.dius.pact' version '4.1.7'
+	id 'au.com.dius.pact' version '4.3.9'
```

Comment:
hi @ishratt - did you check if the serenity report is being generated for this upgraded version?